hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/stream/Release038xStreamType.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.mono.stream;
+
```

Comment:
This is to just validate record streams after replay ?

Reply:
We just need to give the `RecordStreamManager` _some_ `StreamType` to make it happy, I chose this one!